libsolidity/interface/CompilerStack.h

```diff
@@ -336,8 +342,8 @@ class CompilerStack: boost::noncopyable
 	/// @returns a JSON representing the estimated gas usage for contract creation, internal and external functions
 	Json::Value gasEstimates(std::string const& _contractName) const;
 
-	/// Overwrites the release/prerelease flag. Should only be used for testing.
-	void overwriteReleaseFlag(bool release) { m_release = release; }
+	/// Overwrites the metadata format. Should only be used for testing.
```

Comment:
```suggestion
	/// Changes the format of the metadata appended at the end of the bytecode.
	/// This is mostly a workaround to avoid bytecode and gas differences between compiler builds
	/// caused by differences in metadata. Should only be used for testing.
```